megatron/utils.py

```diff
@@ -226,3 +226,19 @@ def flops_calculator(model, args, iteration_time):
     effective_tera_flops_per_gpu = giga_flops_per_model_per_train_step / (iteration_time * 1000.0 * gpus_per_model)
 
     print_rank_0(f"Effective Tera Flops per GPU: {round(effective_tera_flops_per_gpu, 2)} and total parameters {round(approx_parameters_in_billions, 3)} B")
+
+def throughput_calculator(model, args, iteration_time):
```

Comment:
@GuanhuaWang -- is it possible to merge the existing TFLOPs calculation and printing with your new PR?

Please see this: https://github.com/microsoft/Megatron-DeepSpeed/blob/222d899230ce8084699b009cecb492c6108eac1a/megatron/training.py#L792

Reply:
@awan-10 done
[https://github.com/microsoft/Megatron-DeepSpeed/blob/guanhua/fastio/megatron/utils.py#L251](url)